thrift/shared.thrift

```diff
@@ -28,6 +28,10 @@ exception InternalServiceError {
   1: required string message
 }
 
+exception InternalDataCorruptionError {
```

Comment:
should this be called DataCorruption or DataInconsistency ? Almost all of the cases we have are data inconsistency

Reply:
Resolved